src/main/java/org/springframework/samples/petclinic/model/Game.java

```diff
@@ -54,6 +58,18 @@ public class Game extends UUIDEntity {
     @JsonIgnore
     List<Player> raw_players;
 
+    @OneToMany(mappedBy = "game", cascade = CascadeType.ALL, orphanRemoval = true)
+    @Size(max = 8)
+    List<GamePlayer> raw_game_players = new ArrayList<>();
+
+    @OneToOne
+    @JoinColumn(name = "central_card_id")
+    private Card centralCard;
```

Comment:
Para ser consistentes, que luego en frontend es un lío cada una con un nombre distinto.

```suggestion
    private Card central_card;
```